src/queries/a11yValue.ts

```diff
@@ -12,33 +16,38 @@ import type {
 } from './makeQueries';
 import { CommonQueryOptions } from './options';
 
-type A11yValue = {
-  min?: number;
-  max?: number;
-  now?: number;
-  text?: string;
-};
-
 const queryAllByA11yValue = (
   instance: ReactTestInstance
 ): ((
-  value: A11yValue,
+  value: AccessibilityValueMatcher,
   queryOptions?: CommonQueryOptions
 ) => Array<ReactTestInstance>) =>
   function queryAllByA11yValueFn(value, queryOptions) {
     return findAll(
       instance,
       (node) =>
-        typeof node.type === 'string' &&
-        matchObjectProp(node.props.accessibilityValue, value),
+        typeof node.type === 'string' && matchAccessibilityValue(node, value),
       queryOptions
     );
   };
 
-const getMultipleError = (value: A11yValue) =>
-  `Found multiple elements with accessibilityValue: ${JSON.stringify(value)} `;
-const getMissingError = (value: A11yValue) =>
-  `Unable to find an element with accessibilityValue: ${JSON.stringify(value)}`;
+const buildErrorMessage = (value: AccessibilityValueMatcher) => {
```

Comment:
I didn't understand this function at first, I think the naming error is a bit deceiving since it's not really errors at this point but rather the list of all accessibility values that are supposed to match. What about renaming this to something like formatAccessibilityValueMatcherToString ?